services/web/client/source/class/osparc/auth/LoginPage.js

```diff
@@ -133,7 +134,7 @@ qx.Class.define("osparc.auth.LoginPage", {
         versionLinkLayout.add(separator);
       }
 
-      const organizationLink = new osparc.ui.basic.LinkLabel("© 2019 IT'IS Foundation", "https://itis.swiss").set({
+      const organizationLink = new osparc.ui.basic.LinkLabel("© 2020 IT'IS Foundation", "https://itis.swiss").set({
```

Comment:
The build date is also available as an env. could we use it? then we do not need to update this manually

Reply:
Not even need for that. I will use JavaScripts standard built-in date object to print the current year...